layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl/0001-Split-defconfig-into-single-and-dual-version-of-the-.patch

```diff
@@ -0,0 +1,48 @@
+From 19ac56196d5bd182e38b71e79291ad5b0912ed7a Mon Sep 17 00:00:00 2001
+From: Theodor Gherzan <theodor@resin.io>
+Date: Thu, 2 Jun 2016 22:03:20 +0200
+Subject: [PATCH] Split defconfig into single and dual version of the colibri
+ board
+
+Upstream-Status: Pending
```

Comment:
Yes, I would like to push upstream, not exactly sure if this is something desired by the maintainer. Will raise a ticket anyways there.